install_config/configuring_authentication.adoc

```diff
@@ -1720,13 +1718,12 @@ must be allowed to redirect to `<master>/oauth2callback/<identityProviderName>`.
 <6> The client secret. This value may also be provided in an
 xref:../install_config/master_node_configuration.adoc#master-node-configuration-passwords-and-other-data[environment
 variable, external file, or encrypted file].
-<7> Use the value of the `sub` claim in the returned `id_token` as the user's
```

Comment:
the existing text is more accurate

Reply:
I like the new text, but I might move the annotation marker down a line so it's on the (one item) list itself.